spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'liking photos' do
+  before do
+  	@user = User.create email: 'natso@gmail.com', password: '12345678', password_confirmation: '12345678'
+    login_as @user
+    @photo = Photo.create description:'nice', image_file_name:'gramophone.png', user_id: @user.id
```

Comment:
Line is too long. [97/80]<br>Inconsistent indentation detected.<br>Space missing after colon.